test/functional/mempool_datacarrier.py

```
@@ -49,11 +48,11 @@ def run_test(self):
         self.wallet.rescan_utxos()
 
         # By default, only 80 bytes are used for data (+1 for OP_RETURN, +2 for the pushdata opcodes).
-        default_size_data = random_bytes(MAX_OP_RETURN_RELAY - 3)
-        too_long_data = random_bytes(MAX_OP_RETURN_RELAY - 2)
-        small_data = random_bytes(MAX_OP_RETURN_RELAY - 4)
-        one_byte = random_bytes(1)
-        zero_bytes = random_bytes(0)
+        default_size_data = randbytes(MAX_OP_RETURN_RELAY - 3)
```

Ruff (0.15.9)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)